tests/test_config/test_files.py

```diff
@@ -115,44 +115,30 @@ def test_returns_empty_dict(self, caplog: LogCaptureFixture) -> None:
             assert config.read_config_file() == {}
             assert "No configuration file found." in caplog.text
 
+        def test_finds_correct_file_from_multiple_options(self, tmp_path: Path):
+            """If there are multiple config files, the first one with content wins."""
+            setup_cfg = tmp_path / "setup.cfg"
+            setup_cfg.write_text("[metadata]\nname: just-a-name\n", encoding="utf-8")
+            bumpversion_cfg = tmp_path / ".bumpversion.cfg"
+            bumpversion_cfg.write_text("\n", encoding="utf-8")
+            pyproject_toml = tmp_path / "pyproject.toml"
+            pyproject_toml.write_text(
+                "[tool.bumpversion]\n"
+                'current_version = "0.10.5"\n'
+                'parse = "(?P<major>\\\\d+)\\\\.(?P<minor>\\\\d+)\\\\.(?P<patch>\\\\d+)(\\\\-(?P<release>[a-z]+))?"\n'
+                "serialize = [\n"
+                '    "{major}.{minor}.{patch}-{release}",\n'
+                '    "{major}.{minor}.{patch}"\n'
+                "]\n",
+                encoding="utf-8",
+            )
+            with inside_dir(tmp_path):
+                cfg_file = bumpversion.config.files.find_config_file()
+                cfg = config.get_configuration(cfg_file)
 
-@pytest.mark.parametrize(
-    ["conf_file", "expected_file"],
-    [
-        param("basic_cfg.toml", "basic_cfg_expected.json", id="toml basic cfg"),
-    ],
-)
-def test_read_toml_file(conf_file: str, expected_file: str, fixtures_path: Path) -> None:
```

Comment:
This test is no longer needed? Or was it moved?

Reply:
This functionality is better covered by the tests for `read_config_file`. The `read_toml_file` is just a thin wrapper around `tomlkit.parse`.